### DISCUSSION TITLE
[vote ended on 2023-09-13] Generic policy for supported ansible-core versions for community.general

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#271] [Votes done]
This vote is about adopting the following policy for supported ansible-core versions for a major release of community.general:

> For every new major release of community.general, drop support for all ansible-core versions that have been End of Life for at least a couple of weeks before the release date.

What this means concretely is described in https://github.com/ansible-community/community-topics/issues/245#issuecomment-1601585504.

Vote +1 to accept this new policy (which would mean that community.general 8.0.0, the next major release, drops support for ansible-core 2.11 and 2.12), and -1 to reject it.

Please use #245 for discussions, and only post comments with "+1" or "-1" in them here. Thanks.

**Comment by felixfontein** [comment]
+1

**Comment by UnderGreen** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by jamescassell** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by ecchong** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by leogallego** [comment]
+1

**Comment by Andersson007** [comment]
+1
